raiden/transfer/channel.py

```diff
@@ -1583,6 +1585,16 @@ def handle_action_close(
     return TransitionResult(channel_state, events)
 
 
+def handle_action_set_fee(
+        channel_state: NettingChannelState,
+        set_fee: ActionChannelSetFee,
+) -> TransitionResult[NettingChannelState]:
+    msg = 'caller must make sure the ids match'
+    assert channel_state.identifier == set_fee.channel_identifier, msg
+    channel_state.mediation_fee = set_fee.mediation_fee
```

Comment:
Hummm, I think `0` is a valid mediation fee value. Additionally, this is the value used by the current migration, because this is the current implicit value.